cypress/integration/stories/admin-settings/administrators.spec.ts

```diff
@@ -34,36 +38,28 @@ describe('Admin Settings - Administrators Story', () => {
   });
 
   it('should have only one admin', () => {
-    TrafficMonitor.newTrafficMonitor()
-      .method(RequestType.GET)
-      .url(Endpoint.Administrators)
-      .retry(retries)
-      .expect(Response.newResponse(ResponseType.LIST).elements(adminsCount));
+    AdminSettings.AdministratorsPage.verifyAdminCount(adminsCount);
   });
 
   it('should add second admin', () => {
-    AdminSettings.AdministratorsPage.getAddAdminBtn().click();
-    AdminSettings.AdministratorsPage.getAddAdminDialogEmailInput()
-      .type(Config.userEmail())
-      .should(Condition.HaveValue, Config.userEmail());
-    AdminSettings.AdministratorsPage.getAddAdminDialogSaveBtn().click();
+    if (Mocks.enabled()) {
```

Comment:
If the code to add admin is already moved to `addAdmin` method please move also the part that is checking if mocks are enabled there too.